components/CreateEvent.js

```diff
@@ -125,8 +125,12 @@ class CreateEvent extends React.Component {
                   </Link>
                 </p>
               </Container>
-            )}
-            {canCreateEvent && (
+            ) : collective.isFrozen ? (
+              <MessageBox withIcon type="warning" my={5}>
+                <FormattedMessage defaultMessage="This account is currently frozen and cannot be used to create events." />{' '}
+                <FormattedMessage defaultMessage="Please contact your fiscal host for more details." />
```

Comment:
It would be nice to link to the contact form of the fiscal host here since otherwise users might not know how to contact the fiscal host? 🤔